### PR TITLE
MAUI-5256 : Prepare Style UG for .NET MAUI Popup

### PR DESCRIPTION
**Pull request by RiyashameedM**
Prepared Style UG for .NET MAUI Popup

### CODE REVIEW

**Comment by RiyashameedM** [comment]
rebuild

File: MAUI/Popup/styles.md [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+            popup.Show();
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Displaying a .NET MAUI Popup with blur](/Images/styles/maui-popup-with-blurred-background.png)
```
Comment (jayaleshwari) [inline MAUI/Popup/styles.md:426]: Displaying a .NET MAUI Popup with blur overlay

**Review by jayaleshwari — CHANGES_REQUESTED** [review]
Please address review comments

**Review by PiruthivirajM — APPROVED** [review]
approved

File: MAUI/Popup/styles.md (hunk 1 above)
Comment (RiyashameedM) [reply MAUI/Popup/styles.md]: modified

File: MAUI/Popup/styles.md [hunk 2]
```diff
@@ -0,0 +1,588 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Displaying a .NET MAUI Popup with overlay customization](/Images/styles/maui-popup-with-overlay-customization.png)
+
+### Set overlay opacity
+
+The SfPopup allows you adjust the opacity of the overlay color by setting the color value to "#30FF0000". The first two digits represent opacity in hexadecimal format, while the remaining six digits represent the RGB color.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/Popup/styles.md]: modified

**Review by jayaleshwari — APPROVED** [review]
This is fine

**Pull request merged**